src/AI/Muriel.java

```diff
@@ -586,7 +586,7 @@ private void queueUnitProductionActions(GameMap gameMap)
 
       // If we have a lot of cash on hand, don't worry about cost effectiveness - just maximize damage instead.
       // If we ever collect more than twice our income in funds, we just aren't spending fast enough. Fix that.
-      int incomePerTurn = myCo.ownedProperties.size() * myCo.incomePerCity;
+      int incomePerTurn = myCo.ownedProperties.size() * myCo.gameRules.incomePerCity;
```

Comment:
I think we want some sort of "true income" getter for this, because this is just a bug breeding ground